Packs/SophosCentral/Integrations/SophosCentral/README.md

```diff
@@ -13,6 +13,7 @@ The unified console for managing Sophos products.
 | isFetch | Fetch incidents | False |
 | fetch_severity | Fetch Severity | False |
 | fetch_category | Fetch Category | False |
+| fetch_product | Fetch Product | False |
```

Comment:
Please generate only the relevant commands/parameters without delete the command examples.